app/components/sidebars/drawer_layout.js

```diff
@@ -0,0 +1,436 @@
+/* eslint-disable  */
+// https://github.com/react-native-community/react-native-drawer-layout
+// @flow
```

Comment:
this file is a copy from the react-native-drawer-layout library

Reply:
Should we provide credits to the author in the header here?

Reply:
That is why I added the link to the repo in the header.

@jasonblais can you confirm this is enough ?

Reply:
@enahum @sudheerDev Since there is no copyright notice in the original repository, we can add our own to the file and provide the credit to the work in NOTICE.txt.

Reply:
@wiersgallak is also in the NOTICE.txt file

Reply:
@enahum I thought we populate Notice,txt based on package.json with some recent changes hence i was checking with you about licence here in the file.

Reply:
in this PR https://github.com/mattermost/mattermost-mobile/pull/2290/files#diff-d8b7f7a4958ed5af485670e07cf80cacL1436

Reply:
Aah Sorry about that then!

Reply:
are we ok to merge then?

Reply:
If you guys prefer I can fork the repo and used the fork instead

Reply:
For example: 

// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See License.txt for license information.

Reply:
@wiersgallak that is also in the top of this file